app/common/ThemePrefs.ts

```diff
@@ -81,6 +81,7 @@ export const tokensCssMapping = {
   primaryMuted: 'primary-muted',
   primaryDim: 'primary-dim',
   primaryEmphasis: 'primary-emphasis',
+  primaryTransparent: 'primary-transparent',
```

Comment:
I suggest renaming it to "translucent", since "transparent" usually means completely see-through.